cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go

```diff
@@ -1702,6 +1714,30 @@ var InstanceTypes = map[string]*InstanceType{
 		MemoryMb:     12582912,
 		GPU:          0,
 	},
+	"u-18tb1": {
+		InstanceType: "u-18tb1",
+		VCPU:         448,
+		MemoryMb:     0,
```

Comment:
Any reason it get 0 MemoryMb?

Reply:
Ah, I failed to notice that! I think I'll need to dig into this and figure out why that's happening.

Reply:
Finally got around to spending more time on digging into this, and it looks like the Pricing API we're currently using can't be fully trusted. For one thing the correct API designation of these instances is `u-18tb1.metal`, not `u-18tb1` as returned by the pricing API.

I'm going to raise a separate issue to discuss whether/how we move away from the Pricing API as it seems it can't be fully trusted to build a proper picture of these nodes, in the meantime any problem with merging this as is? (I've updated it with a couple more changes generated when running today and double checked them against the info returned by instance descriptions using a boto3 client.)

Reply:
Em.. `u-18tb1.metal` is the right name according to release note. I don't have concern merging this since this instances are available from dedicated hosts with 3 yrs reservation. Most users may not even have access to it. Feel free to open a new issue and let's resolve the pricing api issue there. Some other projects uses nodes template as well like CNI, I will also have a check on reliable API.